e2e/nx-plugin-e2e/tests/plugin-create-nodes.e2e.test.ts

```diff
@@ -158,6 +158,7 @@ describe('nx-plugin', () => {
         configurations: {},
         executor: `@code-pushup/nx-plugin:cli`,
         options: {},
+        parallelism: true,
```

Comment:
Why this?

Reply:
It seems to be expected there by the unit test result

Reply:
Nx 19 added is as default flag that signify that target can be run in parallel (I assume we want to keep it `true`)